app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :likes
```

Comment:
[OPTIONAL] If you are using a has_many connection. I suggest you add `dependent destroy` after it. It helps you to remove all the connected items from the main part. More info: [how to use d.d ](https://stackoverflow.com/questions/29560805/rails-how-to-use-dependent-destroy-in-rails)